### PR TITLE
Introduce livereload for reloading page

### PR DESCRIPTION
**Pull request by rickychien**
It rocks!

### CODE REVIEW

File: _index.html [hunk 1]
```diff
@@ -10,9 +10,13 @@
...
-    <!-- build:backend_js -->
-    <script src="src/backend/main.js"></script>
-    <!-- endbuild -->
+    <!-- build:remove -->
+    <script src="http://localhost:35729/livereload.js"></script>
+    <!-- end:build -->
+
+    <script type="text/javascript">
+      window.requireNode = window.require;
+    </script>
```
Comment (rickychien) [inline _index.html]: Workaround here. I'm not sure what is the best way to do it.

**Comment by rickychien** [comment]
r?=@EragonJ 
Flags: review?(eragonj+moz@eragonj.me)

**Comment by EragonJ** [comment]
There are some bugs in this PR and would break `gulp build`

**Comment by EragonJ** [comment]
@rickychien , I just merged the patch on https://github.com/EragonJ/Kaku/commit/7ce58e56129de7f2e897c0b384d7ecaee29da1f5 , thanks !!

**Pull request closed without merging**